app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_dependency 'solidus_bolt'
+
+module SolidusBolt
+  class BoltConfiguration < ApplicationRecord
+    validate :config_can_be_created, on: :create
+
+    def self.fetch
+      first
```

Comment:
Have you considered to use [first_or_create](https://apidock.com/rails/ActiveRecord/Relation/first_or_create)? It will reduce a lot of complexity in this PR removing the `new`, `create` and `destroy` actions for the controller

Reply:
I'm not sure how it is supposed to remove the `new`, `create` and `destroy` methods because in the end the User has to input the values in the configuration on the `new` page.

And `destroy` is a separate action.

Reply:
The PR has been modified to accomodate the `first_or_create` action for the model